lib/elastomer/client/multi_percolate.rb

```diff
@@ -80,7 +80,8 @@ def initialize(client, params = {})
       #
       # Returns this MultiPercolate instance.
       def percolate(doc, header = {})
-        add_to_actions(:percolate => header)
+        merged = @params.merge(header)
```

Comment:
FWIW, I'd probably call this like `add_to_actions(percolate: @params.merge(header))` since `#merge` returns a new Hash. It's personal preference, though.

Reply:
ah yeah I just did that for clarity of what I was changing but more concise is good I agree, will update!